lib/chef/knife/customization_helper.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Author:: Malte Heidenreich (https://github.com/mheidenr)
+# License:: Apache License, Version 2.0
+#
+
+module CustomizationHelper
+  def CustomizationHelper.wait_for_sysprep(vm, vim_connection, timeout, sleep_time)
```

Comment:
I didn't know you could do it this way, that is, `module Foo; def Foo.bar`. What do you think about this approach  v.s. `module Foo; private; def bar` and then the class that uses it can call `bar` directly instead of `Foo.bar`?